app/src/main/java/com/example/concordia_campus_guide/Adapters/DirectionWrapper.java

```diff
@@ -32,12 +28,16 @@ public void populateAttributesFromStep(DirectionsStep directionStep){
         this.polyline = directionStep.polyline;
 
         //get color is of type transit
-        if(directionStep.travelMode.toString().equals(ClassConstants.TRANSIT)){
+        if (directionStep.travelMode.toString().equals(ClassConstants.TRANSIT)) {
             this.transitDetails = directionStep.transitDetails;
         }
     }
 
-    private String getTransitType(TravelMode travelMode){
+    public DirectionWrapper() {
+        // being used in tests
+    }
```

Comment:
why isn't the default constructor on top of its class? I think it should be the first method

Reply:
I agree w/ you. @CarenRizk do you know why?

Reply:
no particular reason, i guess we just made it after when we were making the tests